### DISCUSSION TITLE
《凉宫春日的串联》汉化笔记

### DISCUSSION THREAD

**Discussion by giscus-xzonn** [Xzonn/xzonn.github.io#18] [General]
# 《凉宫春日的串联》汉化笔记

《凉宫春日的串联》汉化笔记 - 来自Xzonn的小站

https://xzonn.top/posts/Suzumiya-Haruhi-no-Chokuretsu-Chinese-Localization.html

**Comment by Xzonn** [comment]
![汉化招募中！](https://github.com/Xzonn/ChokuretsuChsLocalization/raw/master/assets/images/poster.png)

**Comment by sumuzline** [comment]
sakura13b的机翻效果已经很不错了，只需进行简单的审校，或许可以降低翻译人员的负担

**Reply by Xzonn** [reply]
谢谢，我研究一下。
